app/models/course.rb

```diff
@@ -571,12 +571,7 @@ def student_visa?
 
   def funding_type
     ActiveSupport::Deprecation.new.warn('`funding_type` is deprecated and will be removed. Please use `funding` instead.')
-
-    if FeatureService.enabled?(:db_backed_funding_type)
-      not_set? ? program.funding_type : funding
```

Comment:
We definetly don't have any courses with the `not_set` value?

Reply:
We do not:
```
Course.where(funding: 'not_set').count  # 0
Course.where(funding: ['fee', 'salary', 'apprenticeship']).count # 114869
Course.count # 114869
```